client/me/security-account-recovery/recovery-phone.jsx

```diff
@@ -20,7 +20,7 @@ class RecoveryPhone extends Component {
 			<ManageContact
 				type="sms"
 				isLoading={ isLoading }
-				title={ translate( 'Recovery SMS Number', {
+				title={ translate( 'Recovery SMS number', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.